src/app/views/sidebar/resource-explorer/ResourceLinkV9.tsx

```diff
@@ -0,0 +1,235 @@
+  }
+
+  setExisting(resourceLink, existsInCollection(link, paths, version));
+
+  const handleAddToCollectionClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    event.stopPropagation();
+    props.resourceOptionSelected(ResourceOptions.ADD_TO_COLLECTION, link);
+  }
+
+  const handleRemoveFromCollectionClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    event.stopPropagation();
+    props.resourceOptionSelected(ResourceOptions.REMOVE_FROM_COLLECTION, link);
+  }
+
+  return (
+    <span className={linkStyles.link} tabIndex={0}>
+      <ResourceLinkNameContainer resourceLink={resourceLink} linkStyles={linkStyles} colors={colors} />
+      <ResourceLinkCountBadge resourceLink={resourceLink} />
+      <ResourceLinkActions
+        resourceLink={resourceLink}
+        iconButtonStyles={linkStyles}
+        openDocumentationLink={openDocumentationLink}
+        handleAddToCollectionClick={handleAddToCollectionClick}
+        handleRemoveFromCollectionClick={handleRemoveFromCollectionClick}
+      />
+        &nbsp;
+    </span>
+  );
+}
+
+const ResourceLinkNameContainer = ({
+  resourceLink,
+  linkStyles,
+  colors
+}: {
+  resourceLink: IResourceLink,
+  linkStyles: any,
+  colors: Record<string, Colors>
+}) => (
+  resourceLink.method ? (
+    <span className={linkStyles.resourceLinkNameContainer}>
+      <Badge
+        className={linkStyles.badge}
+        size='medium'
+        color={colors[resourceLink.method]}
+        aria-label={'http method ' + resourceLink.method + ' for'}>
+        {resourceLink.method}
+      </Badge>
+    </span>
+  ) : (
+    <span className={linkStyles.resourceLinkNameContainer}>
+      <span className={linkStyles.resourceLinkText}>
+        {resourceLink.name}
+      </span>
+    </span>
+  )
+);
+
+const ResourceLinkCountBadge = ({ resourceLink }: { resourceLink: IResourceLink }) => (
+  resourceLink.count && resourceLink.count > 0 ? (
+    <Badge appearance='tint' color='informative' aria-label={resourceLink.count + translateMessage('Resources')}>
+      {resourceLink.count}
+    </Badge>
+  ) : null
+);
+
+const ResourceLinkActions = ({
+  resourceLink,
+  iconButtonStyles,
+  openDocumentationLink,
+  handleAddToCollectionClick,
+  handleRemoveFromCollectionClick
+}: {
+  resourceLink: IResourceLink,
+  iconButtonStyles: any,
+  openDocumentationLink: () => void,
+  handleAddToCollectionClick: (event: React.MouseEvent<HTMLButtonElement>) => void,
+  handleRemoveFromCollectionClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+}) => (
+  <div className={`${iconButtonStyles.actions} actions`}>
+    {resourceLink.method && (
+      <Tooltip
+        withArrow
+        content={
+          resourceLink.docLink? translateMessage('Read documentation')
+            : translateMessage('Query documentation not found')
+        }
+        relationship='label'
+      >
+        <Button
```

Comment:
Let's use a disabled button on cases where there's no documentation. Currently on the UI there difference isn't clear